threshold_tools_application_examples.ipynb

```diff
@@ -55,11 +55,6 @@
    },
    "outputs": [],
    "source": [
-    "!pip install -v --upgrade git+https://github.com/cal-adapt/climakitae.git\n",
-    "\n",
-    "#install needed package to calculate parameters for extreme distributions via l-moments\n",
-    "#climakitae includes lmoments 1.0.1, which references scipy functionality which has been depreciated\n",
-    "#command below installs lmoments 1.0.4 which does not make references to missing/depreciated values\n",
     "!pip install git+https://github.com/OpenHydrology/lmoments3.git"
    ]
```

Comment:
This package is not needed by this notebook (or any). Best to delete this import statement.

Reply:
Please also delete from threshold_tools_basics.ipynb